### ISSUE TITLE
Linking with `link.exe` failed, unresolved external symbol __imp_ExtractIconW referenced in function get_app_icon

### ISSUE THREAD

**Issue by lukewilson2002** [ihalila/pdcurses-sys#6]
# Summary
I have been working on this project several months. I have been building the project successfully from this computer and environment (not the same Rust version, but MSVC has not changed.) About a month ago I began having this link-time problem. I have continued to work on the project on my other Linux computer, where it successfully compiles using the GNU Compiler Collection. On a third computer, with Windows 10 and same Rust version, it also failed to build with the same error message. That third computer and this computer both use the Microsoft Build Tools 2017 for C++. I reinstalled them, but the problems persist.

I should mention that I am not directly depending on `pancurses` or `pancurses-sys`, but rather `easycurses` by Lokathor. `easycurses` directly depends on `pancurses`.

# Error Log
On `cargo build`, I get this error:
```
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "/NXCOMPAT" "/LIBPATH:C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.12udk5ebeecrq2jt.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.12wi31b7eo743frk.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.18ezqbgkiegzmnny.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1abprebt92ydmdph.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1b95julfcqi8zix.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1f5ealzg8v1lblcs.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1givpdid60todwv8.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1k0m1vdlqnvssm1h.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1kl3ix2eab1rjzcv.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1otye0dfavaydqqf.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1pzqjohlenx2fu5q.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1uurjpr4b24rj1kk.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1v76zt74x36b7wcb.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1xtm921t52hqam88.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1y2okt2xvbrtlscs.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.1zq3un218c8vthmc.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.20e640gu5sj89n6z.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.21a9h2vr4m48c7lf.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.25jappm2orjsie1w.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.291p9z6gfl3a4f00.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2cjfkhx079s5ye9f.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2kdo778pi88me6um.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2ln5h04kf7s9tg7x.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2nalg85cd7jlwdsz.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2need85jfmm6wfd1.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2o7zktn9nq7lijig.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.2ryjwauy7p6sd400.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.361yz3h669mjpn5r.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.37tr8djl97srguhw.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.3a764646l45r7k28.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.3om48iqd6ahnifg4.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.44cf1ryaq0j897t4.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4a34mkhntywpelkm.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4acy9a0nzc6u94xw.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4cc9q088is7xqs19.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4kfa5setg0v9mojh.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4layyqaln5p5wyzq.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4n9viibtaetv4cw6.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4t65zxfi2k06xeke.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.4tll04wcb5p9snc.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.50y0gsohi805k0mr.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.53pocof41z99x4kw.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.54hgpoodjmxfvtl3.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.57y8ck0w8d5ewajx.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.58aqew8ai4n2e8xn.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.5aheggvv7aqdljir.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.5eidxcs3ek31dnqm.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.fwmxbbbzysiu06i.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.iz05jdmonrttrde.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.qlqee81o8hzaijs.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.xcf6oeps2ooqmau.rcgu.o" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.yrncrnzbaipywes.rcgu.o" "/OUT:C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.exe" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\term3d-a7e98c5afbf57740.rz9uge5rhb6xf5p.rcgu.o" "/OPT:REF,NOICF" "/DEBUG" "/NATVIS:C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/LIBPATH:C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps" "/LIBPATH:C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\build\\pdcurses-sys-7ebf2830ce75c356\\out" "/LIBPATH:C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libordered_float-6c43ec3d5d99b51a.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libnum_traits-da2bd8a6fe558d98.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libeasycurses-d80e59cc5db9041f.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libpancurses-4b3649446c108b8e.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libwinreg-fba70f2178dcf802.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libwinapi-d32e906425b1402c.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libpdcurses-2e91a9c245776ea8.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\liblibc-cadbcc52ccf7c1ff.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\liblog-5fc1604e202bd51e.rlib" "C:\\Users\\Luke\\Documents\\Projects\\term3d\\target\\debug\\deps\\libcfg_if-fc88b54ef5b1401d.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-9f6c449c222c7de8.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-cfe6c4250349ebea.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-6a7d846d0c03a55e.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-86a8e68a6d9428ab.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-381357fb2fbc4c84.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-f73e5492bfaed7b2.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-8885b6a62239bd55.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-8d9516f98ff154f4.rlib" "C:\\Users\\Luke\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-005d97bb518081ca.rlib" "advapi32.lib" "kernel32.lib" "user32.lib" "gdi32.lib" "comdlg32.lib" "advapi32.lib" "ws2_32.lib" "userenv.lib" "msvcrt.lib"
  = note: libpdcurses-2e91a9c245776ea8.rlib(pdcscrn.o) : error LNK2019: unresolved external symbol __imp_ExtractIconW referenced in function get_app_icon
          C:\Users\Luke\Documents\Projects\term3d\target\debug\deps\term3d-a7e98c5afbf57740.exe : fatal error LNK1120: 1 unresolved externals


error: aborting due to previous error

error: Could not compile `term3d`.
```

# Environment
Operating system: Windows 10 Home
Default host: `x86_64-pc-windows-msvc`
Rust version: `rustc 1.33.0-nightly (d99a320cb 2018-12-18)`
MSVC compiler: `Microsoft (R) C/C++ Optimizing Compiler Version 19.16.27025.1 for x64`
MSVC linker: `Microsoft (R) Incremental Linker Version 14.16.27025.1`

Further questions are welcome, thank you for your work on this library. You are free to see the repository my work is held under, if it helps any: https://github.com/asmoaesl/term3d/

**Comment by ronniec95** [comment]
Did anyone solve this? I'm seeing the same problem

**Comment by Jaggernaut555** [comment]
This seems to be fixed by using the source repo of #5. Using easycurses you can replace the dependency by using something such as this in your `cargo.toml`
```
[replace]
"pdcurses-sys:0.7.0" = { git = "https://github.com/zen-chromeless-platform/pdcurses-sys" }
```

**Comment by ihalila** [comment]
I merged #5 and released v0.7.1 with it included, this should be fixed now.
